js/interface.js

```diff
@@ -1761,6 +1763,16 @@ $('input[name="exclude"]').on('tokenfield:createtoken', function(event) {
   });
 });
 
+$('input[id="excludes"]').on('click', function() {
```

Comment:
You can use one listener instead of two if you had named the fields and values correctly.

```suggestion
$('input[name="column-list-mode"]').on('click', function() {
  columnListMode = $(this).val();
  updateSaveRuleValidation();
});
```

Reply:
@tonytlwu  changes made by @NikitaJadhavXoriant  as per suggestion